ex2.js

```diff
@@ -0,0 +1,24 @@
+const printSum = function (arr) {
```

Comment:
```suggestion
const getSum = function (arr) {
```

un nume mai potrivit ar fi asta, pentru ca printarea se face in afara